readme.md

```diff
@@ -195,6 +195,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Open Source Society University](https://github.com/open-source-society/computer-science)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis)
+- [Software Defined Network](https://github.com/sdnds-tw/awesome-sdn)
```

Comment:
According to Wikipedia it's `Software-Defined Networking`

You're also very inconsistent on the usage in your readme. Try to make it more consistent.